weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/objectClassQuery.ts

```diff
@@ -13,17 +13,20 @@ import {
 } from './traceServerClientTypes';
 import {Loadable} from './wfDataModelHooksInterface';
 
-type BaseObjectClassRegistry = typeof baseObjectClassRegistry;
-type BaseObjectClassRegistryKeys = keyof BaseObjectClassRegistry;
-type BaseObjectClassType<C extends BaseObjectClassRegistryKeys> = z.infer<
-  BaseObjectClassRegistry[C]
+type BuiltinObjectClassRegistry = typeof builtinObjectClassRegistry;
+type BuiltinObjectClassRegistryKeys = keyof BuiltinObjectClassRegistry;
+type BaseObjectClassType<C extends BuiltinObjectClassRegistryKeys> = z.infer<
+  BuiltinObjectClassRegistry[C]
 >;
 
 export type TraceObjSchemaForBaseObjectClass<
-  C extends BaseObjectClassRegistryKeys
+  C extends BuiltinObjectClassRegistryKeys
 > = TraceObjSchema<BaseObjectClassType<C>, C>;
 
-export const useBaseObjectInstances = <C extends BaseObjectClassRegistryKeys>(
+// Notice: this is still `base` object class, not `builtin` object class.
```

Comment:
Future readers might prefer a descriptive comment rather than this notice which feels more relevant for reviewers (which I appreciate)